sql/world/guildhouse_spawns.sql

```diff
@@ -1,68 +1,140 @@
+	(19187, 16231.1, 16295.6, 13.1759, 3.02647, 'Leatherworking Trainer (Master)'),
+	(28696, 16228.9, 16304.7, 13.1819, 4.64831, 'Skinning Trainer (Grand Master)'),
+	(28703, 16222.9, 16304.7, 13.1821, 4.56585, 'Alchemy Trainer (Grand Master)'),
+	(19052, 16220.5, 16302.3, 13.176, 6.14647, 'Alchemy Trainer (Master)'),
+	(28704, 16218.3, 16284.3, 13.1756, 6.1975, 'Herbalism Trainer (Grand Master)'),
+	(28699, 16220.2, 16299.6, 13.178, 6.22894, 'Tailoring Trainer (Grand Master)'),
+	(28706, 16225, 16310.9, 29.262, 6.22119, 'First Aid Trainer (Grand Master)'),
+	(28742, 16225.3, 16313.9, 29.262, 6.28231, 'Fishing Trainer (Grand Master)'),
+	(33587, 16256.3, 16291.7, 13.174, 2.22426, 'Cooking Trainer (Grand Master)'),
+    (6491, 16319.937, 16242.404, 24.4747, 2.206830, 'Spirit Healer'),
+	(30605, 16228.0, 16280.5, 13.1761, 2.98877, 'Banker'),
+	(8719, 16238.1, 16291.9, 22.9303, 1.46240, 'Alliance Auctioneer'),
+	(9856, 16242.7, 16291.3, 22.9318, 1.45454, 'Horde Auctioneer'),
+	(9858, 16240.5, 16291.5, 22.9318, 1.47417, 'Neutral Auctioneer'),
+	(184137, 16220.3, 16272, 12.9736, 4.45592, 'Mailbox (Object)'),
+	(187293, 16230.5, 16283.5, 13.9061, 3, 'Guild Vault (Object)'),
+    (191028, 16255.5, 16304.9, 20.9785, 2.97516, 'Barber Chair (Object)'),
+	(1685, 16257.3, 16298.8, 13.1758, 3.14050, 'Forge (Object)'),
+	(4087, 16254.5, 16299.8, 13.1758, 6.19537, 'Anvil (Object)'),
+	(59853, 16257.6, 16293.0, 13.17508, 2.989519, 'Stove (Object)'),
+	(500000, 16232.9, 16264.1, 13.55570, 3.028813, 'Portal: Stormwind (Object)'),
+	(500001, 16232.8, 16257.1, 13.93456, 3.028813, 'Portal: Darnassus (Object)'),
+	(500002, 16231.3, 16254.2, 13.65647, 3.028813, 'Portal: Exodar (Object)'),
+	(500003, 16233.4, 16260.6, 13.84770, 3.028813, 'Portal: Ironforge (Object)'),
+	(500004, 16232.9, 16264.1, 13.55570, 3.028813, 'Portal: Orgrimmar (Object)'),
+	(500005, 16231.3, 16254.2, 13.65647, 3.028813, 'Portal: Silvermoon (Object)'),
+	(500006, 16233.4, 16260.6, 13.84770, 3.028813, 'Portal: Thunder Bluff (Object)'),
+	(500007, 16232.8, 16257.1, 13.93456, 3.028813, 'Portal: Undercity (Object)'),
+    (500008, 16211.1, 16266.9, 13.7458, 5.6724, 'Portal: Shattrath (Object)'),
+	(500009, 16213.9, 16270.5, 13.1378, 5.4996, 'Portal: Dalaran (Object)'),
+    (28690, 16226.8, 16269.4, 13.0858, 3.88255, 'Stable Master'),
+	(28692, 16236.2, 16315.7, 20.8454, 4.64365, 'Trade Supplies'),
+	(28776, 16223.7, 16297.9, 20.8454, 6.17044, 'Tabard Vendor'),
+	(4255, 16230.2, 16316.1, 20.8455, 4.64365, 'Food & Drink Vendor'),
+    (29636, 16233.2, 16315.9, 20.8454, 4.64365, 'Reagent Vendor'),
+    (29493, 16229.1, 16286.4, 13.176, 3.03831, 'Ammo & Repair Vendor'),
+	(2622, 16242.8, 16302.1, 13.176, 4.55570, 'Poisons Vendor');
+
+/*
+-- Updates existing creatures to the new creatures.
+-- Replace <world> and <characters> with your database names!
+
+-- The following only needs to be ran if you used this module before 5-1-2022.
```

Comment:
I don't know why this was added? Seems like we shouldn't refer back to almost two years ago?